handler/src/main/java/io/netty/handler/ssl/SniHandler.java

```diff
@@ -102,10 +113,19 @@ public String hostname() {
     }
 
     /**
+     * @return the selected {@link Supplier} for {@link SslContext}
+     */
+    protected Supplier<? extends SslContext> sslContextSupplier() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove usage of generic wildcard type. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1452)